src/mpid/ch4/netmod/ofi/init_settings.c

```diff
@@ -256,13 +276,17 @@ void MPIDI_OFI_init_settings(MPIDI_OFI_capabilities_t * p_settings, const char *
     UPDATE_SETTING_BY_CAP(major_version, MPIR_CVAR_CH4_OFI_MAJOR_VERSION);
     UPDATE_SETTING_BY_CAP(minor_version, MPIR_CVAR_CH4_OFI_MINOR_VERSION);
     UPDATE_SETTING_BY_CAP(num_am_buffers, MPIR_CVAR_CH4_OFI_NUM_AM_BUFFERS);
+    if (!strcmp(prov_name, "sockets")) {
```

Comment:
Is it necessary to limit to "sockets". Is there provider advertize wrong capability, e.g. enable_triggereed but actually doesn't support triggering?

Reply:
I think the original intention was to limit to sockets because that is the only provider that currently supports triggered ops. However, I think the capability check should be sufficient. Will remove checking prov_name against sockets

Reply:
Since testing has passed and reviews are complete, I'll push off removing the prov_name check in a separate PR